.gitignore

```diff
@@ -21,6 +21,7 @@ deployment/ansible/roles/azavea.*
 # Django
 /src/django/static/
 /src/django/data/
+/venv
```

Comment:
Minor, but I use a hidden `.venv` since the folder rarely needs to be looked in to.

I also have `.venv` ignored in my global `~/.gitignore`, but adding it here makes sense too.